src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,125 @@
 
-A project information icon is located to the right of the string overview. Translators can view information regarding the project, its priority level, and testing by clicking the icon.
+The profile menu is located at the far right of the main toolbar. Clicking the profile image will reveal a dropdown menu where users can perform several actions, like navigate to their [profile page](profile.md), [download and upload translations](translate.md#downloading-and-uploading-translations), etc.
 
-The notifications icon, represented by a bell, is located on the right side of the main toolbar. Translators will be notified of any important updates regarding the current project. By clicking the bell icon, translators can view notifications in more detail or see all active notifications.
+![Profile menu](../../assets/images/pontoon/ui/profile_menu.png "Screenshot of the profile menu")
 
-The profile menu is located at the far right of the main toolbar. Clicking the profile image will reveal the drop-down menu. From this menu, translators can perform several actions, like navigate to their profile page, download and upload translations, etc. For information about uploading and downloading translation check [this document](translate.md#downloading-and-uploading-translations), for Machinery see [here](resources.md#machinery-tab).
+Note that some profile menu items are only available to users with specific [permissions](users.md#user-roles).
 
-![Profile menu](../../assets/images/pontoon/ui/profile_menu.png)
+### String list and filters
 
-Note that some profile menu items are restricted by [user rights](users.md#user-roles).
+The sidebar displays the list of strings in the current project resource. For each string there is:
+* A colored square that identifies the string status (i.e. *Missing*, *Translated*, etc.).
+* The source string.
+* The approved translation or the most recent suggestion if available.
 
-### Sidebar
-
-The sidebar displays the list of strings in the current project resource. Each string is displayed with the string status (i.e. *Missing*, *Translated*, etc.) identified by a colored square, the source string, and the approved translation or the most recent suggestion if available.
-
-![Sidebar](../../assets/images/pontoon/ui/sidebar.png)
+![Sidebar](../../assets/images/pontoon/ui/sidebar.png "Screenshot of the sidebar, with a list of strings showing the different string statuses")
 
 Color legend:
 * **<span style="color: #7bc876;">green</span>**: string is translated.
-* **<span style="color: #fed271;">yellow</span>**: string is fuzzy.
+* **<span style="color: #c0ff00;">light-green</span>**: string is pretranslated.
 * **<span style="color: #ffa10f;">orange</span>**: string has warnings.
 * **<span style="color: #f36;">red</span>**: string has errors.
 * **<span style="color: #4d5967;">gray</span>**: string is missing.
```

Comment:
Should we use "string" or "translation" here? For the first two, translation is a no-no, but for the last one (or three), it makes more sense.